docs/exp-algo/leiden.md

```
@@ -0,0 +1,87 @@
+
+This implementation runs inside FalkorDB’s UDF runtime and uses `graph.traverse` to build an in-memory adjacency representation of the supplied node set.
+
+Compared to `exp.louvain`, Leiden adds a **refinement** step. In this implementation, refinement is done by splitting each community into **connected components**, which helps avoid disconnected communities.
+
+## Syntax
+```cypher
+flex.exp.leiden({nodes: nodes, resolution: 1, maxLevels: 10, maxPasses: 10, seed: 42})
+```
+
+## Parameters
+| Parameter | Type | Required | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)